.github/workflows/prettier.yml

```
@@ -0,0 +1,34 @@
+  pull_request:
+permissions:
+  contents: write
+jobs:
+  prettier:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          token: ${{ secrets.GITHUB_TOKEN }} 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)